.github/workflows/issue-automation.yml

```
@@ -21,7 +20,7 @@ jobs:
         with:
           script: |
             const issue = context.payload.issue;
-
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)